Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/MeshSensor_ROSSensorConnection.cs

```diff
@@ -148,6 +150,11 @@ void Update()
         }
     }
 
+    public string GetSensorName()
+    {
+        return sensorName;
```

Comment:
ditto with the return this.gameObject.name;

you can add a setter if you want as well.

Also worth noting that idk if gameObjects can change their names...

Reply:
gameObjects can change their names btw